### PR TITLE
Documentation: release with project key

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

**Comment by gyuho** [comment]
/cc @xiang90 @dghubble 

Thanks.

File: Documentation/dev/release.md [hunk 1]
```diff
@@ -61,16 +60,14 @@ It generates all release binaries and images under directory ./release.
...
 
 The following commands are used for public release sign:
 
 ```
 cd release
-# personal GPG is okay for now
-for i in etcd-*{.zip,.tar.gz}; do gpg --sign ${i}; done
-# use `CoreOS ACI Builder <release@coreos.com>` secret key
-gpg -u 88182190 -a --output etcd-${VERSION}-linux-amd64.aci.asc --detach-sig etcd-${VERSION}-linux-amd64.aci
+for i in etcd-*{.zip,.tar.gz}; do gpg2 --default-key $SUBKEYID --output ${i}.asc --detach-sign ${i}; done
```
Comment (xiang90) [inline Documentation/dev/release.md]: have you tried these commands? where is $SUBKEYID defined?

**Comment by gyuho** [comment]
Yeah the command works. And the subkey is the key ID of our project key. You can only see when you connect that key to your machine and do pgp2 card status.

**Comment by xiang90** [comment]
@gyuho Probably doc that a little bit?

**Comment by dghubble** [comment]
General instructions for all teams or for your reference. https://github.com/coreos-inc/security/blob/master/openpgp/aci-signing.md Feel free to improve them if you run into issues.

File: Documentation/dev/release.md (hunk 1 above)
Comment (gyuho) [inline Documentation/dev/release.md]: Just added more explanation. As Dalton explained, the whole process is documented in our private repository.

Thanks.

**Comment by xiang90** [comment]
lgtm

**Comment by gyuho** [comment]
Will merge right before we release the 2.3. Want to make sure the whole command work again.

**Comment by philips** [comment]
The key is documented here: https://coreos.com/security/app-signing-key/

**Pull request merged**